assets/js/vue/img.js

```diff
@@ -33,30 +34,55 @@ export default {
         this.canGenerateSrcSet() ? `no-small img-responsive` : '',
       ];
     },
+    isCloudinary() {
+      return Tools.isTrue(this.cloudinary);
+    },
     source() {
-      return Tools.isTrue(this.cloudinary) ? this.noCloudinary || this.getCloudinaryLink(this.img) : this.noCloudinary;
+      return this.isCloudinary ? this.noCloudinary || this.getCloudinaryLink(this.img) : this.noCloudinary;
     },
     loading() {
       return this.lazy ? 'lazy' : null;
     },
     crossOriginValue() {
-      return Tools.isTrue(this.cloudinary) ? (this.crossorigin ? this.crossorigin : 'anonymous') : null;
+      return this.isCloudinary ? (this.crossorigin ? this.crossorigin : 'anonymous') : null;
     },
     isSvg() {
       return this.img?.indexOf('.svg') !== -1;
     },
+    hasPictureTag() {
+      return this.isCloudinary && this.imgSrcSets;
+    },
+    pictureWrapperClassList() {
+      return ['img__picture-wrapper', this.imgSrcSetValue?.ratioClasses];
+    },
+    imgSrcSetValue() {
+      return ImgSrcSets[this.imgSrcSets];
+    },
+    imgSrcSetSources() {
+      return this.imgSrcSetValue?.srcSets?.filter((item) => item.media);
+    },
+    imgSrcSetImg() {
+      const srcSets = this.imgSrcSetValue?.srcSets;
+
+      if (!srcSets) return null;
+
+      return this.getSourceLink(srcSets[srcSets.length - 1]);
+    },
   },
   created() {
     if (this.canGenerateSrcSet()) return;
 
-    if (Tools.isTrue(this.cloudinary)) return;
+    if (this.isCloudinary) return;
 
     this.noCloudinary = this.getBaseAssetPath();
     this.sizes = DefaultPresets.sizes;
   },
   methods: {
+    getSourceLink(srcSet) {
+      return `${basePath}${srcSet.params}${this.img}`;
```

Comment:
Wieso kombiniert man diese Methode nicht mit getCloudinaryLinkWithTransformation oder getCloudinaryBasePathLink in etwas universelles?